packages/sync/src/modules/Terms.php

```diff
@@ -12,6 +12,23 @@ function name() {
 		return 'terms';
 	}
 
+	/**
+	 * Allows WordPress.com servers to retrieve a term object via the sync API.
+	 *
+	 * @param string $object_type The type of object.
+	 * @param int $id The id of the object.
+	 *
+	 * @return bool|\WP_Term
+	 */
+	public function get_object_by_id( $object_type, $id ) {
+		if ( $object_type === 'term' ) {
+			$term = get_term( intval( $id ) );
+			return $term ? $term : false;
```

Comment:
Actually, on a second thought, I think the `$term` check might not be enough. `get_term()` can return a `WP_Error` in some cases, so should we make this check `$term && ! is_wp_error( $term )`?

Reply:
```suggestion
			return ( $term && ! is_wp_error( $term ) ) ? $term : false;
```